keyboards/heliar/wm1_hotswap/rules.mk

```diff
@@ -0,0 +1,41 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```

Comment:
Lines 14-23 can be deleted. QMK sets this based on the MCU and BOOTLOADER rules.

https://github.com/qmk/qmk_firmware/blob/3a7085dee4389bfc3356336180b44d294cf0738a/bootloader.mk#L34-L43
https://github.com/qmk/qmk_firmware/blob/3a7085dee4389bfc3356336180b44d294cf0738a/bootloader.mk#L86-L88